core/src/main/java/org/mskcc/cbio/oncokb/model/clinicalTrialsMathcing/Arm.java

```diff
@@ -5,7 +5,7 @@
 /**
  * Created by Yifu Yao on 2020-09-08
  */
-public class Arms {
+public class Arm {
     private String armDescription;

```

Comment:
I believe `armDescription` is required because it has to match the schema outputted by the trials endpoint for the data to be returned. Otherwise null values will be returned